serverless/openvino/omz/intel/person-reidentification-retail-300/nuclio/main.py

```diff
@@ -0,0 +1,31 @@
+import json
+import base64
+from PIL import Image
+import io
+from model_handler import ModelHandler
+
+def init_context(context):
+    context.logger.info("Init context...  0%")
+
+    model = ModelHandler()
```

Comment:
Codacy found an issue: [No value for argument 'labels' in constructor call](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)